src/rdtime.h

```diff
@@ -195,7 +195,13 @@ static RD_INLINE void rd_timeout_init_timespec_us (struct timespec *tspec,
                 tspec->tv_sec = timeout_us;
                 tspec->tv_nsec = 0;
         } else {
+                #if __APPLE__
```

Comment:
Unindent #... to the leftmost column

Reply:
done. Also changed the if to ifdef for consistency.

Reply:
also rebased.